docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def _prune_candidates(T, m, min_dist, is_cands,  T_sliding_mu = None, T_sliding_std = None):
</pre><p>Can you use&nbsp;<code>M_T</code>&nbsp;and&nbsp;<code>Σ_T</code>&nbsp;please? And then make them required parameters and not optional. So, any parent function must precompute them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>